src/pages/Calculator.tsx

```diff
@@ -123,9 +131,11 @@ export const Calculator = (): React.ReactElement => {
     return { points: expectedValue, lowerBound, upperBound }
   }, [calculatorData, setQuery])
 
+  // Location mode and not blank OR fully valid manual mode
```

Comment:
Could you either make the two sides of this OR statement follow the same pattern or remove it?

// Location mode and location chosen OR manual mode and filled out.

Reply:
removed